packages/tailwindcss/src/utilities.ts

```diff
@@ -2261,122 +2261,205 @@ export function createUtilities(theme: Theme) {
+      },
+    ])
 
-      if (!candidate.value) {
-        return [
-          decl('--tw-gradient-position', `in oklch,`),
-          decl('background-image', `conic-gradient(var(--tw-gradient-stops))`),
-        ]
-      }
+    function handleBgConic({ negative }: { negative: boolean }) {
+      return (candidate: Extract<Candidate, { kind: 'functional' }>) => {
+        if (candidate.value?.kind === 'arbitrary') {
+          if (candidate.modifier) return
+          let value = candidate.value.value
+          return [
+            decl('--tw-gradient-position', `${value},`),
+            decl('background-image', `conic-gradient(var(--tw-gradient-stops,${value}))`),
+          ]
+        }
 
-      let value = candidate.value.value
+        let interpolationMethod = resolveInterpolationModifier(candidate.modifier)
+
+        if (!candidate.value) {
+          return [
+            decl('--tw-gradient-position', `${interpolationMethod},`),
+            decl('background-image', `conic-gradient(var(--tw-gradient-stops))`),
+          ]
+        }
+
+        let value = candidate.value.value
 
-      if (candidate.value.kind === 'arbitrary') {
-        return [
-          decl('--tw-gradient-position', `${value},`),
-          decl('background-image', `conic-gradient(var(--tw-gradient-stops,${value}))`),
-        ]
-      } else {
         if (!isPositiveInteger(value)) return
 
         value = negative ? `calc(${value} * -1)` : `${value}deg`
 
         return [
-          decl('--tw-gradient-position', `from ${value} in oklch,`),
+          decl('--tw-gradient-position', `from ${value} ${interpolationMethod},`),
           decl('background-image', `conic-gradient(var(--tw-gradient-stops))`),
         ]
       }
     }
-  }
 
-  utilities.functional('-bg-conic', handleBgConic({ negative: true }))
-  utilities.functional('bg-conic', handleBgConic({ negative: false }))
+    utilities.functional('-bg-conic', handleBgConic({ negative: true }))
+    utilities.functional('bg-conic', handleBgConic({ negative: false }))
 
-  utilities.functional('bg-radial', (candidate) => {
-    if (candidate.modifier) return
+    suggest('bg-conic', () => [
+      {
+        hasDefaultValue: true,
+        modifiers: suggestedModifiers,
+      },
+      {
+        values: ['0', '30', '60', '90', '120', '150', '180', '210', '240', '270', '300', '330'],
+        supportsNegative: true,
+        modifiers: suggestedModifiers,
+      },
+    ])
 
-    if (!candidate.value) {
-      return [
-        decl('--tw-gradient-position', `in oklch,`),
-        decl('background-image', `radial-gradient(var(--tw-gradient-stops))`),
-      ]
-    }
+    utilities.functional('bg-radial', (candidate) => {
+      if (!candidate.value) {
+        let interpolationMethod = resolveInterpolationModifier(candidate.modifier)
+        return [
+          decl('--tw-gradient-position', `${interpolationMethod},`),
+          decl('background-image', `radial-gradient(var(--tw-gradient-stops))`),
```

Comment:
also no fallback here so I guess we just rm the one on line 2396: https://github.com/tailwindlabs/tailwindcss/pull/14984/files?diff=split&w=1#diff-036c40df8e1498a5c399fb971238a19fcdab257001b9703661a46a110984b56aR2396

Reply:
Same thing as the conic one — values like `radial-gradient(red, blue)` are valid so it's useful to just have a `bg-radial` utility that specifies no position/direction but `radial-gradient(in oklch)` isn't valid so the fallback isn't useful.